_posts/2017-12-12-my-experience-at-js-crash-course.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: My experience at the JavaScript Crash Course Berlin
+date: 2017-12-12
+categories: story
+---
+
+I recently had the opportunity to attend the [JavaScript Crash Course Berlin](http://wtmberlin.com/javascript-crash-course/).
+It was a 2 months course, the lectures were every Thursday from 7 PM to 10 PM and the goal of the course was 
```

Comment:
looks like you have some new lines slipped in the middle of this sentence and a few others :)

Reply:
Yes, that's true! Fixed! 👍